packages/create-cloudflare/e2e-tests/cli.test.ts

```diff
@@ -258,5 +258,98 @@ describe.skipIf(frameworkToTest || isQuarantineMode())(
 				expect(output).toContain(`type API starter (OpenAPI compliant)`);
 			},
 		);
+
+		test.skipIf(process.platform === "win32")(
+			"Going back and forth between the category, type, framework and lang prompts",
+			async () => {
+				const { output } = await runC3(
+					[projectPath, "--git=false", "--no-deploy"],
+					[
+						{
+							matcher: /What would you like to start with\?/,
+							input: {
+								type: "select",
+								target: "Demo application",
+							},
+						},
+						{
+							matcher: /Which template would you like to use\?/,
+							input: {
+								type: "select",
+								target: "Queue consumer & producer Worker",
+							},
+						},
+						{
+							matcher: /Which language do you want to use\?/,
+							input: {
+								type: "select",
+								target: "Go back",
+							},
+						},
+						{
+							matcher: /● Queue consumer & producer Worker/,
```

Comment:
Is this matcher not meant to match a prompt question? What does it mean to match an active option and then select "Go back"?

Reply:
Yes. It is intentional to confirm that we have the previous selected option as the default value.

Reply:
Makes sense. But it's a bit implicit. If this tests fails in the future, I'm not sure the dev would know why this is the matcher.

We should either leave a comment on each of these matchers as-is or extend the PromptHandler config to match multiple regexes so we can test for the question _and_ this active element

Reply:
I have updated the select prompt handler to support a `defaultValue` option.